### PR TITLE
Update React and dev dependencies

### PR DESCRIPTION
**Pull request by jnbt**
Updates this project to use:

* Latest React (15.0.1)
* babel instead of jsx-loader (as jsx-loader is deprecated)
* ES2015 as the preferred way to use React

/cc @headjump @splattael

### CODE REVIEW

File: test/test.js [hunk 1]
```diff
@@ -1 +1 @@
-require("app_test");
\ No newline at end of file
+  require("app_test")
```
Comment (splattael) [inline test/test.js]: @jnbt Are those 2 spaces mandatory?

**Comment by splattael** [comment]
LGTM!

File: test/test.js (hunk 1 above)
Comment (jnbt) [inline test/test.js]: :art:

**Comment by SolidNerd** [comment]
Should we upgrade React to 15.0.1 ?

**Comment by jnbt** [comment]
Sure makes sense. I'll give it a try.

**Comment by jnbt** [comment]
Works for me using 15.0.1

**Pull request merged**